src/index.css

```diff
@@ -0,0 +1,7 @@
+* {
+  box-sizing: border-box;
+}
```

Comment:
I'm a bit concerned by the "*" here - won't this set the value for every single element on the page? I think we could take it out if not required, to not have any unintended knock-on effects.

Reply:
https://www.w3schools.com/css/css3_box-sizing.asp
![Screenshot 2023-10-30 at 13 34 08](https://github.com/iO-Academy/23-sept-catmode-bookswap-fe/assets/144483187/1a34cac6-be6a-457e-b5bc-7ef604ace38d)

Reply:
Ok, that's fine, we'll keep it in - thanks for the supporting info!